frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
```

Comment:
this file can be removed

Reply:
removed

Reply:
deleted this file and imports

Reply:
@mcraepv Sorry, where would I find the imports?